Parsers/ASimAuthentication/ProductParsers/AuthenticationM365D.yaml

```diff
@@ -0,0 +1,131 @@
+Parser:
+  Title: ASIM M365 Defender DeviceLogonEvent Authentication parser
+  Version: '0.1.0'
+  LastUpdated: July 1, 2021
+Product:
+  Name: M365 Defender
+Normalization:
+  Schema: Authentication
+  Version: '0.1.0'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/AzSentinelAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AzSentinelNormalization
+Description: |
+      This Query Parser maps M365 Defender Device File Events (DeviceFileEvents) to the Azure Sentinel Information Model authenticaion schema.
+ParserName: vimAuthenticationM365Defender
+ParserQuery: |
+  let FaliureReason=datatable(EventOriginalResultDetails:string, EventResultDetails:string)[
+    'InvalidUserNameOrPassword','No such user or password' 
+    // What other will we find here?
+    ];
+  let AuthM365D=(){
+    DeviceLogonEvents 
+    // ---   Already mapped
+    // // TimeGenerated - exists
+    //
+    | project-rename 
+       EventOriginalResultDetails=FailureReason 
+       , EventSubType=LogonType
+    | extend 
+    //
+       // ----  Untouched
+       TimeGenerated
+    // 
+      // ----  Event
+       , EventCount=int(1)
+       , EventStartTime=TimeGenerated
+       , EventEndTime=TimeGenerated
+       , EventOriginalType = EventSubType // Previously:LogonType
+       , EventProduct='M365 Defender for EndPoint'
+      ,  EventResult = case(ActionType =='LogonSuccess', 'Success'
+                            , ActionType=='LogonFailed', 'Failure'
+                            , ActionType=='LogonAttempted', 'NA' //// Not sure what this is
+                            , 'NA')
+      , EventSchemaVersion='0.1.0'
+      , EventType='Logon'
+      , EventVendor ='Microsoft'
+    //    -   No Data
+    //  EventOriginalUid 
+    //  EventProductVersion 
+    //  EventReportUrl
+    //  EventMessage 
+    //  
+    //  
+    //  ----   Target and Actor Users
+     | project-rename 
+         TargetUserId=AccountSid
+       , ActorUserId =InitiatingProcessAccountSid
+       , ActorUserUpn=InitiatingProcessAccountUpn
+       , ActorUserObjectId=InitiatingProcessAccountObjectId
+      | extend 
+         TargetUserIdType ='UserSid'
```

Comment:
Type is "SID" not "UserSid"

Reply:
yup! also need to fix documentation: 
https://docs.microsoft.com/en-us/azure/sentinel/normalization#the-device-entity
identifiers for a user:UserId